packages/mask/popups/pages/Trader/Header.tsx

```diff
@@ -0,0 +1,67 @@
+import { Icons } from '@masknet/icons'
+import { RoutePaths, useTrade } from '@masknet/plugin-trader'
+import { PopupRoutes } from '@masknet/shared-base'
+import { MaskTabList } from '@masknet/theme'
+import { TabContext } from '@mui/lab'
+import { Tab } from '@mui/material'
+import { useLocation, useNavigate } from 'react-router-dom'
+import { NormalHeader } from '../../components/index.js'
+
+export function Header() {
+    const { mode } = useTrade()
+    const basepath = '/trader'
+    const titleMap: Record<RoutePaths, string | null> = {
+        [RoutePaths.Trade]: 'Exchange',
+        [RoutePaths.History]: 'History',
+        [RoutePaths.Confirm]: 'Confirm Swap',
+        [RoutePaths.BridgeConfirm]: 'Confirm Bridge',
+        [RoutePaths.SelectLiquidity]: 'Select Liquidity',
+        [RoutePaths.Slippage]: 'Slippage',
+        [RoutePaths.QuoteRoute]: 'Quote Route',
+        [RoutePaths.BridgeQuoteRoute]: 'Quote Route',
+        [RoutePaths.TradingRoute]: 'Trading Route',
+        [RoutePaths.Exit]: null,
+        [RoutePaths.NetworkFee]: 'Network fee',
```

Comment:
Skip translation here until we merge it into develop branch, and we will do it with `lingui`.